src/libs/blueprint/conduit_blueprint_mesh_matset_xforms.cpp

```diff
@@ -1425,6 +1426,263 @@ to_multi_buffer_by_material(const conduit::Node &src_matset,
 // -- end conduit::blueprint::mesh::matset --
 //-----------------------------------------------------------------------------
 
+//-----------------------------------------------------------------------------
+// -- begin conduit::blueprint::mesh::specset --
+//-----------------------------------------------------------------------------
+namespace specset
+{
+//-----------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+void
+to_silo(const conduit::Node &specset,
+        const conduit::Node &matset,
+        conduit::Node &dest)
+{
+    if(!specset.dtype().is_object() )
+    {
+        CONDUIT_ERROR("blueprint::mesh::specset::to_silo passed specset node"
+                      " must be a valid specset tree.");
+    }
+
+    if(!matset.dtype().is_object() )
+    {
+        CONDUIT_ERROR("blueprint::mesh::specset::to_silo passed matset node"
+                      " must be a valid matset tree.");
+        // TODO well, not entirely true...
```

Comment:
I want to give users the option to just pass the silo representation of the matset directly into this function if they already have one. Or they can pass a regular matset. Either way, this function will be smart and create the silo matset if it has to. So this error message is somewhat dishonest... should I change it? Also, is that even a good idea?

Reply:
Yes good idea but change message